logrotate/config.sls

```diff
@@ -22,3 +32,33 @@ logrotate-directory:
     - mode: 755
     - makedirs: True
 
+{% if ns.hourly == True %}
```

Comment:
```suggestion
{%- if ns.hourly %}
```

The above is sufficient.  Note also the use of Jinja whitespace control (`{%- ... %}`).  Again, there have been a number of discussions about this and we've found this structure to be effective in nearly all situations.  There are rare cases where you need to apply the post-whitespace control (`{% ... -%}`) but we're agreed that we should never use pre- and post-whitespace control (`{%- ... -%}`).

One of the main benefits of this is working through the debug output, showing how states have been rendered.  It is also very useful in templates.